corehq/apps/accounting/bootstrap/utils.py

```diff
@@ -108,20 +108,19 @@ def _ensure_features(feature_rates, edition, verbose, apps):
 
     features = []
     for feature_type in feature_rates.keys():
-        if feature_type not in FeatureType.EDITIONED_FEATURES:
-            feature = Feature(name=feature_type, feature_type=feature_type)
-        else:
-            feature = Feature(name=f"{feature_type} {edition}", feature_type=feature_type)
+        feature_name = feature_type
+        if feature_type in FeatureType.EDITIONED_FEATURES:
+            feature_name = f"{feature_name} {edition}"
             if edition == SoftwarePlanEdition.ENTERPRISE:
-                feature.name = f"Dimagi Only {feature.name}"
+                feature_name = f"Dimagi Only {feature_name}"
```

Comment:
with the nested `if`s i find it easier to read the following...

```
if feature_type in FeatureType.EDITIONED_FEATURES:
            feature_name = (f"Dimagi Only {feature_name}" if edition == SoftwarePlanEdition.ENTERPRISE
                                         else f"{feature_name} {edition}")
```

Reply:
otherwise, having the `else:` after the second nested if helps contain the nest...
and then assign `feature_name = feature_type` there

Reply:
I don't personally love the long inline conditionals like that but happy to add an `else` if it seems clearer.